middleware/csrf/csrf.go

```diff
@@ -18,8 +19,10 @@ var (
 	ErrRefererNoMatch  = errors.New("referer does not match host and is not a trusted origin")
 	ErrOriginInvalid   = errors.New("origin invalid")
 	ErrOriginNoMatch   = errors.New("origin does not match host and is not a trusted origin")
-	errOriginNotFound  = errors.New("origin not supplied or is null") // internal error, will not be returned to the user
-	dummyValue         = []byte{'+'}
+	ErrNotGetStorage   = errors.New("unable to retrieve data from CSRF storage")
```

Comment:
We need to ensure error naming continuity and also consider including an error for when the token cannot be saved to storage.

Error naming continuity: Err + [Subject] + [Error Condition]

```suggestion
	ErrStorageRetrievalFailed = errors.New("unable to retrieve data from CSRF storage")
	ErrStorageSaveFailed      = errors.New("unable to save data to CSRF storage")
```

Additionally, make sure to apply these changes in the relevant *.manager.go file(s).

Reply:
done in local.